server/src/lib/server.rs

```diff
@@ -189,28 +188,21 @@ fn slot_to_bdf(slot: api::Slot, ty: SlotType) -> Result<pci::Bdf> {
 
 #[endpoint {
     method = PUT,
-    path = "/instances/{instance_id}",
+    path = "/instances",
```

Comment:
Does the plural `instances` make sense to preserve, considering propolis is designed to only ever run a single instance at once?

Reply:
Updated to `instance` everywhere

Reply:
Thanks for cleaning this all up.

Reply:
Missing this one in the last commit?

Reply:
Okay, *now* updated to `instance` everywhere